### ISSUE TITLE
Website search bar does not work

### ISSUE THREAD

**Issue by babobski** [twbs/bootstrap#39521]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Since the new search bar is implemented I can never find what I'm searching for. 
I always end up going trough the whole documentation to find it, or use google to find the correct documentation. 

### Reduced test cases

For now I was searching for the .btn-close documentation, but when you search for btn-close or close I don't end up on the documentation for .btn-close 

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

5.3

**Comment by mdo** [comment]
Yeah, we don't index the classes or markup, so you might not get a match like that. Using the component name and page title is much better. Might be something we can improve down the line though.
